Makefile

```diff
@@ -48,6 +48,14 @@ ifeq ($(STATIC),1)
 GOFLAGS  += -a -tags netgo -ldflags '-extldflags "-lm -lstdc++ -static"'
 endif
 
+LDFLAGS := 
+ifeq ($(RELEASE),1)
+LDFLAGS := -X github.com/cockroachdb/cockroach/util.buildSHA "$(shell git rev-parse HEAD)"
```

Comment:
Nevermind, I see that it's behind the RELEASE flag. Although if we only did this in the `build` target, it might be nice to have this information even for non-release builds.